docker-compose.yml

```diff
@@ -34,7 +34,7 @@ services:
   # https://go-vela.github.io/docs/administration/server/
   server:
     container_name: server
-    image: target/vela-server:latest
+    image: server:local
```

Comment:
suggestion: not sure if this was intended to be checked in, but i'd personally prefer to revert this. it assumes a certain work style. for example, if i want to pair local dev with certain server code, i temporarily point this at the local server repo to live build the image. leaving it as it was will typically allow folks to get the local stack running without having to follow extra steps.